eval/run_fewshot_comparison.py

```
@@ -0,0 +1,174 @@
+    parser.add_argument("--max-eval", type=int, default=15)
+    parser.add_argument("--k", type=int, default=5)
+    parser.add_argument("--out", default="fewshot_comparison_results.json")
+    args = parser.parse_args()
+
+    if not os.environ.get("OPENAI_API_KEY"):
+        print("ERROR: OPENAI_API_KEY not set.")
+        sys.exit(1)
+
+    quizzes = {args.quiz: QUIZ_CONFIGS[args.quiz]} if args.quiz else QUIZ_CONFIGS
+
+    print(f"\nRunning few-shot calibration comparison")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)